pkg/specgenutil/util.go

```diff
@@ -273,6 +273,11 @@ func CreateExitCommandArgs(storageConfig storageTypes.StoreOptions, config *conf
 		return nil, err
 	}
 
+	transient := "--transient-store=false"
```

Comment:
How about
transient:= fmt.Sprintf("--transient-store=%s", strconv.FormatBool(storageConfig.TransientStore))

Reply:
Can do that inline even.